### ISSUE TITLE
Sessions list does not seem to be keyboard accessible

### ISSUE THREAD

**Issue by pwang347** [microsoft/vscode#291060]
Testing #290636

I was unable to tab to individual sessions and start them from keyboard only

<img width="821" height="184" alt="Image" src="https://github.com/user-attachments/assets/470f2f47-f803-4698-aa9a-cb009fc0ee6b" />
